bundler/lib/bundler/rubygems_ext.rb

```diff
@@ -2,6 +2,8 @@
 
 require "pathname"
 
+require "rubygems" unless defined?(Gem)
```

Comment:
Is the defined check faster than the library loaded check? I've not often seen a require guarded like this.